docs/modules/ROOT/pages/migration-7/jdbcusers.adoc

```diff
@@ -0,0 +1,59 @@
+= JDBC-based user storage
```

Comment:
This is the migration documentation. JdbcUserDetailsManager is mentioned elsewhere in the documentation, too. Would you like the updates for that in this PR, or as a separate PR?

Reply:
Ideally, yes, documentation gets updated at the same time. If you are not able, I can add in a polish commit.

Reply:
I can do it. I have just run into teams that were... strict... with PR size. Easier to ask than to break up a too large PR after the fact.

Reply:
Uploaded